requirements-test.txt

```diff
@@ -13,3 +13,4 @@ openwisp-monitoring @ https://github.com/openwisp/openwisp-monitoring/tarball/1.
 django-redis~=5.4.0
 mock-ssh-server~=0.9.1
 channels_redis~=4.2.1
+openwisp-utils[selenium]
```

Comment:
This file already includes `openwisp-utils` at line number 8. You can update that to include selenium extras.